packages/subscription/src/chainable-subscription.ts

```diff
@@ -1,10 +1,22 @@
-import { Operation } from 'effection';
+import { Operation, resource } from 'effection';
 import { Subscription } from './subscription';
 import { DeepPartial, matcher } from './match';
+import { SubscriptionSource, subscribe } from './subscribable';
+
+const DUMMY = { next() { throw new Error('dummy') } };
```

Comment:
This is a bit of a hack, but it was the only way I could come up with which doesn't also weaken the types of `ChainableSubscription` by making the inner subscription optional.